### PR TITLE
Add fix for Callout moving out of bounds from MapView

### PR DESCRIPTION
**Pull request by puneet-pdx**
Add fix for Callout showing at wrong location when MapView is drawn inside a Column or a Row

### CODE REVIEW

File: microapps/MapViewCalloutApp/app/src/main/java/com/arcgismaps/toolkit/mapviewcalloutapp/screens/TapLocationScreen.kt [hunk 1]
```diff
@@ -75,32 +74,30 @@ fun TapLocationScreen(viewModel: MapViewModel) {
...
+            onLongPress = { viewModel.clearMapPoint() },
+            content = if (mapPoint != null && calloutVisibility) {
+                {
+                    Callout(
+                        modifier = Modifier.size(175.dp, 75.dp),
+                        location = mapPoint,
+                        rotateOffsetWithGeoView = rotateOffsetWithGeoView,
+                        offset = offset
+                    )
                     {
```
Comment (sorenoid) [inline microapps/MapViewCalloutApp/app/src/main/java/com/arcgismaps/toolkit/mapviewcalloutapp/screens/TapLocationScreen.kt:92]: formatting seems off here.

```suggestion
                    ) {
```

**Review by sorenoid — APPROVED** [review]
great minimal solution. nice work.

File: toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/MapView.kt [hunk 2]
```diff
@@ -170,28 +173,36 @@ public fun MapView(
...
-            it.geometryEditor = geometryEditor
-            it.grid = grid
-            it.backgroundGrid = backgroundGrid
-            it.isAttributionBarVisible = isAttributionBarVisible
-            it.setTimeExtent(timeExtent)
-            if (it.graphicsOverlays != graphicsOverlays) {
-                it.graphicsOverlays.apply {
-                    clear()
-                    addAll(graphicsOverlays)
+    Box(modifier = modifier.clipToBounds()) {
```
Comment (puneet-pdx) [inline toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/MapView.kt:176]: This ensures that the Callout will not be drawn outside the MapView bounds.

**Review by eri9000 — APPROVED** [review]
LGTM 👍🏼

**Comment by puneet-pdx** [comment]
Thanks @sorenoid @eri9000

**Pull request merged**